[2023-07-05 23:03:38,836] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: API_ETL.conexion_BD manual__2023-07-05T23:03:32.430437+00:00 [queued]>
[2023-07-05 23:03:38,844] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: API_ETL.conexion_BD manual__2023-07-05T23:03:32.430437+00:00 [queued]>
[2023-07-05 23:03:38,845] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-05 23:03:38,845] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-07-05 23:03:38,846] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-05 23:03:38,857] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): conexion_BD> on 2023-07-05 23:03:32.430437+00:00
[2023-07-05 23:03:38,863] {standard_task_runner.py:52} INFO - Started process 195 to run task
[2023-07-05 23:03:38,868] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'API_ETL', 'conexion_BD', 'manual__2023-07-05T23:03:32.430437+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/API_ETL.py', '--cfg-path', '/tmp/tmpelv1sekb', '--error-file', '/tmp/tmpzn4qxcsu']
[2023-07-05 23:03:38,870] {standard_task_runner.py:80} INFO - Job 58: Subtask conexion_BD
[2023-07-05 23:03:38,941] {task_command.py:371} INFO - Running <TaskInstance: API_ETL.conexion_BD manual__2023-07-05T23:03:32.430437+00:00 [running]> on host 0808ffb1fb61
[2023-07-05 23:03:38,979] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-07-05 23:03:39,016] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DylanKaplan
AIRFLOW_CTX_DAG_ID=API_ETL
AIRFLOW_CTX_TASK_ID=conexion_BD
AIRFLOW_CTX_EXECUTION_DATE=2023-07-05T23:03:32.430437+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-05T23:03:32.430437+00:00
[2023-07-05 23:03:39,018] {logging_mixin.py:115} INFO - Conectandose a la BD en la fecha: 2023-07-05 23
[2023-07-05 23:03:40,400] {logging_mixin.py:115} INFO - <connection object at 0xffff82b1f590; dsn: 'user=kaplan46611_coderhouse password=xxx dbname=data-engineer-database host=data-engineer-cluster.cyhh5bfevlmn.us-east-1.redshift.amazonaws.com port=5439', closed: 0>
[2023-07-05 23:03:40,402] {logging_mixin.py:115} INFO - Connected to Redshift successfully!
[2023-07-05 23:03:40,404] {python.py:173} INFO - Done. Returned value was: None
[2023-07-05 23:03:40,432] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=API_ETL, task_id=conexion_BD, execution_date=20230705T230332, start_date=20230705T230338, end_date=20230705T230340
[2023-07-05 23:03:40,458] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-05 23:03:40,495] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
